x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/StaticPlot.js

```diff
@@ -53,8 +54,9 @@ class StaticPlot extends PureComponent {
             curve={'curveMonotoneX'}
             data={serie.data}
             color={serie.color}
-            stroke={serie.color}
+            stroke={serie.stack ? 'rgba(0,0,0,0)' : serie.color}
```

Comment:
stroke has to be transparent for a stacked area chart, because we stick a line chart on top of it to achieve the look that we want.